plugins/deployed-labeler/main.go

```diff
@@ -169,7 +171,7 @@ func (s *server) updatePullRequests(prs []pullRequest, team string) (teamDeploye
 
 		lblTeam := teamLabel(team)
 		if _, belongs := pr.Labels[lblTeam]; !belongs {
-			s.log.Infof("PR %v does not belong to %v, skipping it", pr.Number, team)
+			s.log.Debugf("PR %v does not belong to %v, skipping it", pr.Number, team)
```

Comment:
Why set all those log lines to debug? 😅

Reply:
Moved them back as this info seems useful